docs/configuration.rst

```diff
@@ -109,6 +109,8 @@ Nginx is used to route web traffic to the various applications and to serve stat
 
 MySQL
 *****
+.. note::
+    When configuring an external MySQL database, please make sure it is using version 5.7.
```

Comment:
Could you move this note further down? There is a dedicated paragraph on running a separate MySQL database, so I think you should move it after the list of config variables (i.e: line 126).

Reply:
No problem, moved the note to after the envs portion